### PR TITLE
Provide GitHub templates for new issues and PRs #197

### PR DESCRIPTION
**Pull request by djelinek**
Signed-off-by: Dominik Jelinek <djelinek@redhat.com>

### CODE REVIEW

**Comment by djelinek** [comment]
Related to #197

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+### Environment
+
+- Apache Camel Language Server release version date: e.g. "**1.0.0-SNAPSHOT**, **Jun-04**" ?
+- OS:
+    - [ ] Linux (*RHEL*, *Fedora*, *Ubuntu*, ...)
+    - [ ] Windows
+    - [ ] Mac OS
+- Apache Camel Client for:
+    - [ ] Eclipse
+    - [ ] VS Code
```
Comment (apupier) [inline .github/ISSUE_TEMPLATE.md]: there is also Eclipse Che and OpenShift.io

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+- [ ] Are there **Unit tests**?
+- [ ] Are there **Integration tests**?
+- [ ] Do we need a new **UI test**?
+
+## PR workflow progress
+
+1. [ ] Tagged with relevant **PR labels**
+2. [ ] Green **job for PR**
+3. [ ] PR was created more then **24 hours ago**
```
Comment (apupier) [inline .github/PULL_REQUEST_TEMPLATE.md]: typo then -> than

Comment (apupier) [inline .github/PULL_REQUEST_TEMPLATE.md]: rules is in fact more precisely:
- created more than 24 hours ago or all committers approved it

**Comment by djelinek** [comment]
Hi @apupier, thank you for your review. I processed all your requests. Do you have any other ideas how to improve these templates?

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
```
Comment (apupier) [inline .github/ISSUE_TEMPLATE.md]: since you started to write it, there is now "CodeReady Workspaces" to add

**Pull request merged**